test/e2e/middleware-general/app/pages/api/edge-search-params.js

```diff
@@ -0,0 +1,10 @@
+import { NextResponse } from 'next/server'
```

Comment:
This file creates a new chunk and fails some tests: https://github.com/vercel/next.js/runs/8206300570?check_suite_focus=true#step:9:2358